webapp/channels/src/components/admin_console/ip_filtering/ip_filtering.scss

```diff
@@ -0,0 +1,406 @@
+                        }
+
+                        .FilterName {
+                            width: 291px;
+                        }
+
+                        .IpAddressRange {
+                            flex: 1 1 0;
+                        }
+
+                        .Actions {
+                            // height: 32px;
+                            display: flex;
+                            align-items: center;
+                            justify-content: center;
+
+                            >div {
+                                display: flex;
+                                align-items: center;
+                                justify-content: center;
+                                padding: 8px;
+                                border-radius: 4px;
+
+                                &:hover {
+                                    background: var(--center-channel-color-8, rgba(63, 67, 80, 0.08));
+                                }
+
+                                &.edit {
+                                    color: var(--center-channel-color-72, rgba(63, 67, 80, 0.72));
+                                }
+
+                                &.delete {
+                                    color: var(--error-text, rgba(210, 75, 78, 1));
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            .NoFilters {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                justify-content: center;
+                padding: 32px 20px;
+
+                .Title {
+                    color: var(--center-channel-color, #3f4350);
+                    font-family: Metropolis;
+                    font-size: 20px;
+                    font-style: normal;
+                    font-weight: 600;
+                    line-height: 28px;
+                    text-align: center;
+                }
+
+                .Subtitle {
+                    max-width: 320px;
+                    margin-top: 8px;
+                    color: var(--center-channel-color-72, rgba(63, 67, 80, 0.72));
+                    font-family: Open Sans;
+                    font-size: 14px;
+                    font-style: normal;
+                    font-weight: 400;
+                    line-height: 20px;
+                    text-align: center;
+
+                    .Button {
+                        height: 14px;
+                        padding: 0;
+                        background: none;
+                        color: var(--link-color, #1c58d9);
+                        font-family: Open Sans;
+                        font-size: 14px;
+                        font-style: normal;
+                        font-weight: 400;
+                    }
+                }
+            }
+
+            .AllowedIPAddressesSection {
+                display: inline-flex;
```

Comment:
you shouldn't need any of these styles for this section:

~~overflow: hidden;~~
~~width: 920px;~~
~~background: white;~~
~~border-top-left-radius: 4px;~~
~~border-top-right-radius: 4px;~~